i18n/translations/en/common.json

```diff
@@ -41,7 +41,7 @@
     "phone": "Enter a valid telephone number.",
     "regex": "Enter the correct format.",
     "file-size-too-large": "Select a smaller file.",
-    "file-size-too-large-all-files": "Files exceed the 6 MB maximum. Upload smaller files.",
+    "file-size-too-large-all-files": "Files exceed the 3.5 MB maximum. Upload smaller files.",
```

Comment:
Would be nice to pass in the variable for max vs continuing to update this string

Reply:
Actually let's do that in a separate PR

Reply:
will do later